src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test.Common/CSharpTestLspServerHelpers.cs

```diff
@@ -9,6 +9,7 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+
```

Comment:
Can this be removed?

Reply:
When I formatted documents it kept inserting these in. I think it was something with the particular version of VSMain, because it's not doing it this morning after I've updated.

Reply:
Weird. Should be controlled by https://github.com/dotnet/razor/blob/main/.editorconfig#L40